### ISSUE TITLE
Create a 'Response' model to store user responses. 

### ISSUE THREAD

**Issue by t-walker** [t-walker/RSO-Survey#28]
We might not commit them to the database but we need a Response model which

has_many answers and then every answer that is selected on the screen should be stored

**Comment by t-walker** [comment]
@czhanacek what's the status of this one? It looks like Nick was assigned to the other issue, so should he just take over this or do you have code to commit on this?

**Comment by czhanacek** [comment]
I'm starting it now since I think I just finished #13

**Comment by czhanacek** [comment]
And I thought I had assigned #13 to myself... But I guess I didn't. So @seattlecoder9 sorry in advance if I stepped on your toes :|

**Comment by seattlecoder9** [comment]
What's my next move then @t-walker? Yeah, it's alright 😅 I'll get another task, I'm sure.

**Comment by t-walker** [comment]
We also need unit tests for all of the basic functions involved @czhanacek.

**Comment by t-walker** [comment]
@seattlecoder9 I created another task and assigned you to it.

**Comment by seattlecoder9** [comment]
It seems identical to Charlie's task he just started after finishing my last task.

On Jul 16, 2017 11:20 AM, Tyler Walker <notifications@github.com> wrote:

@seattlecoder9<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_seattlecoder9&d=DwMCaQ&c=C3yme8gMkxg_ihJNXS06ZyWk4EJm8LdrrvxQb-Je7sw&r=J-rx7BbBAbQp-wmtAlcFhn3XKFfhuDYEYSqEbgXEuAA&m=hGOMEN_bvROo7_VSQKanZBDdFSlCohq3c-wm_O8h9tA&s=vIWpf2Qktj0V59jHjK1xHkmHIUHuR4Y81c4gOqITfsw&e=> I created another task and assigned you to it.

**Comment by czhanacek** [comment]
@seattlecoder9 I'll commit what I've got so far and then it's all yours

**Comment by t-walker** [comment]
woops

**Comment by czhanacek** [comment]
@seattlecoder9 committed. Now you've gotta figure out how to convert the form data from the survey into a response.

**Comment by czhanacek** [comment]
I'm confused. Is this still my task or is it Nick's?

**Comment by t-walker** [comment]
It's Nick's now.

**Comment by czhanacek** [comment]
Okay

**Comment by t-walker** [comment]
@czhanacek @seattlecoder9  once this sprint is done (not including testing) we should have a call to go over all of the major functionality as it stands now so that we can come up with something to send to ASWSU

**Comment by t-walker** [comment]
Ideally, I'd like to have something to show to them tomorrow.

**Comment by czhanacek** [comment]
Okay.

**Comment by t-walker** [comment]
@seattlecoder9 what's the status of this?

**Comment by t-walker** [comment]
@seattlecoder9 status?
